opencga-master/src/test/java/org/opencb/opencga/master/monitor/daemons/ExecutionDaemonTest.java

```diff
@@ -418,10 +420,17 @@ public void testRegisterFilesSuccessfully() throws Exception {
         // Check jobId is properly populated
         OpenCGAResult<org.opencb.opencga.core.models.file.File> files = catalogManager.getFileManager().search(studyFqn,
                 new Query(FileDBAdaptor.QueryParams.JOB_ID.key(), job.getId()), FileManager.INCLUDE_FILE_URI_PATH, token);
-        assertEquals(7, files.getNumResults());
+        List<String> expectedFiles = Arrays.asList(
+                outDir,
+                outDir + "file1.txt",
+                outDir + "file2.txt",
+                outDir + "A/",
```

Comment:
@pfurio , in this case, the folder "A/" is created with type "FILE" instead of "DIRECTORY"

Reply:
Just fixed that. Thanks for the report :)